### PR TITLE
Add Firecracker error on getting-started.md

### PR DESCRIPTION
**Pull request by xavinux**
Error may appears when there isn't enough memory for Firecracker to run.

Signed-off-by: xavinux <xavinux@gmail.com>

Issue #835

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -191,3 +191,12 @@ Possible mitigations are:
   kernel so as to use `vmalloc` instead of `kmalloc` for them.
 - Reduce memory pressure on the host.
 
+### Error appears when starting Firecracker.
```
Comment (aghecenco) [inline FAQ.md]: nit: maybe rephrase so as to document more errors that can appear when starting Firecracker.

```suggestion
### Firecracker fails to start.
```

File: FAQ.md [hunk 2]
```diff
@@ -191,3 +191,12 @@ Possible mitigations are:
...
+
+```
+Cannot create VMM.: Vm(VmFd(Error(12)))
```
Comment (aghecenco) [inline FAQ.md]: I think the error message has changed. 75457bfee948da0de6d4adc0c4376467d0abc614 has removed the punctuation sign and the underlying error type. I believe the error message is now:

```
Cannot create VMM: Error opening VM fd: VmFd(Os { code: 12, kind: Other, message: "Out of memory" })
```

I would not use the full error message in the FAQ. I'd rather say something in the lines of `If Firecracker exits logging an error with code X meaning Y, the root cause is Z`. In this case it would be code `12` meaning `Out of memory` with the root cause `not enough memory to run a microVM`.

**Comment by dianpopa** [comment]
Closing this due to inactivity. @xavinux feel free to reopen when the case.
